docs/topics/practices.rst

```diff
@@ -190,11 +190,10 @@ Dynamic Creation of Item Classes
 ================================
 
 For applications in which the structure of item class is to be determined by
-user input, or other changing conditions, you can dynamically create item
-classes instead of manually coding them.
-
-::
+user input or other changing conditions you can return regular Python 
+dicts from spiders.
```

Comment:
A good catch. I just removed the whole section. I don't think we should document workarounds for limitations of older Scrapy versions.